wger/gym/templates/gym/partial_user_list.html

```diff
@@ -6,49 +6,112 @@
+    });
+
+    $('#myTab a').on('click', function (e) {
+      e.preventDefault()
+      $(this).tab('show')
+    })
 });
 </script>
 
-<table class="table table-hover" id="main_member_list">
-<thead>
-<tr>
-    {% for key in user_table.keys %}
-        <th>{{ key }}</th>
-    {% endfor %}
-</tr>
-</thead>
-<tbody>
-{% for current_user in user_table.users %}
-<tr>
-    <td>
-        {{current_user.obj.pk}}
-    </td>
-    <td>
-        <a href="{% url 'core:user:overview' current_user.obj.pk %}">{{current_user.obj}}</a>
-    </td>
-    <td>
-        {{current_user.obj.get_full_name}}
-    </td>
-    <td data-order="{{current_user.last_log|date:'U'}}">
-        {{current_user.last_log|default:'-/-'}}
-    </td>
-    {% if show_gym %}
-    <td>
-        {% if current_user.obj.userprofile.gym_id %}
-            <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
-            {{ current_user.obj.userprofile.gym }}
-            </a>
-        {% else %}
-            -/-
-        {% endif %}
-    </td>
-    {% endif %}
-</tr>
-{% endfor %}
-</tbody>
-</table>
+<ul class="nav nav-tabs">
+    <li class="active"><a data-toggle="tab" href="#active-users">Active Users</a></li>
+    <li><a data-toggle="tab" href="#inactive-users">Deactivated Users</a></li>
+</ul>
+
+<div class="tab-content mb-3">
+    <div id="active-users" class="tab-pane fade in active">
+        <table class="table table-hover" id="active_member_list">
+            <thead>
+            <tr>
+                {% for key in user_table.keys %}
+                    <th>{{ key }}</th>
+                {% endfor %}
+            </tr>
+            </thead>
+            <tbody>
+            {% for current_user in user_table.users %}
+            {% if current_user.obj.is_active %}
+            <tr>
+                <td>
+                    {{current_user.obj.pk}}
+                </td>
+                <td>
+                    <a href="{% url 'core:user:overview' current_user.obj.pk %}">{{current_user.obj}}</a>
+                </td>
+                <td>
+                    {{current_user.obj.get_full_name}}
+                </td>
+                <td data-order="{{current_user.last_log|date:'U'}}">
+                    {{current_user.last_log|default:'-/-'}}
+                </td>
+                {% if show_gym %}
+                <td>
+                    {% if current_user.obj.userprofile.gym_id %}
+                        <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
+                        {{ current_user.obj.userprofile.gym }}
+                        </a>
+                    {% else %}
+                        -/-
+                    {% endif %}
+                </td>
+                {% endif %}
+            </tr>
+            {% endif %}
+            {% endfor %}
+            </tbody>
+        </table>
+    </div>
+    <div id="inactive-users" class="tab-pane fade">
+        <table class="table table-hover" id="inactive_member_list">
```

Comment:
Good work. The feature works well locally. Why don't you consider reusing the table instead of making two tables?

Reply:
We are using data tables and I want to enable the search to work better. And also we want one tab to have rows of active and the other rows of inactive. The third reason was it destroys the HTML structure after reusing the table and the tables tag won't close.
<img width="770" alt="Screenshot 2019-06-18 at 10 45 57" src="https://user-images.githubusercontent.com/45054246/59663004-ca8fae00-91b6-11e9-9e8e-54ddf7f93b0d.png">

Reply:
Cool. Nice work with the descriptive PR message